--- a/test_Google.py
+++ b/test_Google.py
@@ -1,9 +1,65 @@
 def func(param0: int = 10, param1 = "string") -> bool:
-"""docstring
-"""
+    '''This is summary.
+
+    This is more detailed description.
+
+    Args:
+        param0 (int): the description of param0,
+                    a lot of space. balabalabalabalabalabblalblablalblalbalblalb
+                    lablalblalblabllablalabalababalalbalblalblalblallabalabaend
+        param1: the description of param0, it is very long balabalabalabalabalab
+            blalblablalblalbalblalblablalblalblabllablalabalababalalbalblalblalb
+            lallabalabaend
+        test: it is also very long balabalbabalabalabalabalabalabalabalabalabala
+            babalabalbaballbalblalbalblalblalbalblalblablalblalblallblalbalblalb
+            albalblalblaend
+
+    Returns:
+        bool: true for success, false for failure
+        true for success, false for failure
+
+    Raise:
+        IOException: it is also very long balabalbabalabalabalabalabalabalabalab
+            alabalababalabalbaballbalblalbalblalblalbalblalblablalblalblallblalb
+            alblalbalbalblalblaend
+
+    Note:
+        PEP 484_ type annotations are supported. If attribute, parameter, and re
+            turn types are annotated according to PEP 484_, they do not need to 
+            be included in the docstring
+
+    '''
 pass 
 def func(param0: int = 10, param1 = "string") -> bool:
-"""docstring
-"""
+    '''This is summary.
+
+    This is more detailed description.
+
+    Args:
+        param0 (int): the description of param0,
+                    a lot of space. balabalabalabalabalabblalblablalblalbalblalb
+                    lablalblalblabllablalabalababalalbalblalblalblallabalabaend
+        param1: the description of param0, it is very long balabalabalabalabalab
+            blalblablalblalbalblalblablalblalblabllablalabalababalalbalblalblalb
+            lallabalabaend
+        test: it is also very long balabalbabalabalabalabalabalabalabalabalabala
+            babalabalbaballbalblalbalblalblalbalblalblablalblalblallblalbalblalb
+            albalblalblaend
+
+    Returns:
+        bool: true for success, false for failure
+        true for success, false for failure
+
+    Raise:
+        IOException: it is also very long balabalbabalabalabalabalabalabalabalab
+            alabalababalabalbaballbalblalbalblalblalbalblalblablalblalblallblalb
+            alblalbalbalblalblaend
+
+    Note:
+        PEP 484_ type annotations are supported. If attribute, parameter, and re
+            turn types are annotated according to PEP 484_, they do not need to 
+            be included in the docstring
+
+    '''
 pass
